cypress/integration/hosts/host_filtering.ts

```diff
@@ -156,98 +156,121 @@ describe("Hosts page filtering from URL", () => {
   });
 });
 
-describe("Hosts page filtering from table filters", () => {
-  before(() => {
-    cy.login();
-  });
+describe(
+  "Hosts page filtering from table filters",
+  { scrollBehavior: false },
+  () => {
+    before(() => {
+      cy.login();
+    });
 
-  beforeEach(() => {
-    cy.preserveCookies();
-    cy.visit(`${hostsRoute}?limit=100&page=0`);
-    cy.dataCy("hosts-table").should("be.visible");
-    cy.dataCy("hosts-table").should("not.have.attr", "data-loading", "true");
-  });
+    beforeEach(() => {
+      cy.viewport(1920, 1080);
+      cy.preserveCookies();
+      cy.setCookie("This is an important notification", "true");
+      cy.visit(`${hostsRoute}?limit=100&page=0`);
+      cy.dataCy("hosts-table").should("be.visible");
+      cy.dataCy("hosts-table").should("not.have.attr", "data-loading", "true");
+      cy.dataCy(distroFilterIconDataCy)
+        .should("be.visible")
+        .should("not.be.disabled");
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.wait(1000);
```

Comment:
Same comment about `cy.wait(1000)`

Reply:
I agree with @minnakt  using `cy.wait` is an anti pattern and isn't recommended by cypress. https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting

Reply:
I was able to get these tests to pass on this [patch](https://spruce.mongodb.com/task/spruce_ubuntu1604_e2e_test_patch_6c09c39220d1a3d8444d8782696e5d5cba647be3_62e03962c9ec445e89bb1f32_22_07_26_18_58_51/tests?execution=0&sortBy=STATUS&sortDir=ASC) by waiting all of the queries and guarding all of the popovers and inputs. I think waiting all of the queries is important for some reason but I don't know really know why it allows them to pass 🤔

Reply:
It makes sense to guard any queries that could cause the table to rerender and cause the icon to detach. It's possible that one of the queries waited on prevented cypress from accessing the icon before it detaches but it's also possible that waiting on queries caused the timeout to increase like cy.wait(1000)